src/dsl.jl

```diff
@@ -286,112 +277,88 @@ end
 ### DSL Internal Master Function ###
 
 # Function for creating a ReactionSystem structure (used by the @reaction_network macro).
-function make_reaction_system(ex::Expr; name = :(gensym(:ReactionSystem)))
```

Comment:
name symbol generation is now handled where the user (don't) provide the name as input.